### PR TITLE
cli options improvements

### PR DESCRIPTION
**Pull request by agutoli**
## What has been implemented?
conponents cli improvements

## Examples 

Screen Shot 2019-05-12 at 7 01 22 pm
Screen Shot 2019-05-12 at 7 01 41 pm
Screen Shot 2019-05-12 at 7 01 57 pm
Screen Shot 2019-05-12 at 7 02 16 pm
Screen Shot 2019-05-12 at 7 03 09 pm

### CODE REVIEW

File: bin/cli [hunk 1]
```diff
@@ -3,20 +3,52 @@
 /*
  * Serverless Components – Command Line Interface
  */
+const path = require('path');
+const { ArgumentParser } = require('argparse');
+const { version } = require(path.resolve(path.join(__dirname, '../package.json')));
```
Comment (eahefnawy) [inline bin/cli]: thanks for that!

File: bin/cli [hunk 2]
```diff
@@ -3,20 +3,52 @@
...
+  version: version,
+  addHelp:true,
+  prog: "components",
+  description: 'Serverless components usage'
+});
+
+parser.addArgument('method', {
+  action: 'store',
+  nargs: "?",
+  choices: ['deploy', 'remove'],
```
Comment (eahefnawy) [inline bin/cli]: hmmm what do these choices refer to exactly? The components core is pretty agnostic when it comes to components methods. Each component only needs the `default` method as a requirement. All else is optional. So not sure how `deploy` fits in here?

Am I missing something? 🤔

Comment (agutoli) [reply bin/cli]: This is just an example, we can just remove it and keep it agnostic. The central idea is refactoring it to use a cli library.

Comment (agutoli) [reply bin/cli]: @eahefnawy I just remove the options.

**Comment by eahefnawy** [comment]
LG2M

Thanks @agutoli :raised_hands:

**Pull request merged**